src/core/createLogger.js

```diff
@@ -57,6 +57,18 @@ export default ({ getDebugEnabled, console, getMonitors, context }) => {
         data.options
       );
     },
+    logOnCommandResolved(data) {
+      notifyMonitors("onCommandResolved", data);
```

Comment:
Nice! @jonsnyder , do we need to update the public wiki?

Reply:
Yes.